msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -133,11 +133,13 @@ The following table provides the response codes:
 
 ## Send notification signal API
 
-All users in a meeting receive the notifications sent through the `NotificationSignal` API. `NotificationSignal` API enables you to provide meeting signals that are delivered using the existing conversation notification API for user-bot chat. You can send signal based on user action, an in-meeting dialog box. The API includes query parameter, examples, and response codes.
+All users in a meeting receive the notifications sent through the `NotificationSignal` API. `NotificationSignal` API triggers an in-meeting dialog box and enables you to provide meeting signals that are delivered using the existing conversation notification API for user-bot chat. You can send signal based on user action, which is an in-meeting dialog box. The API includes query parameter, examples, and response codes.
```

Comment:
> The API includes query parameter, examples, and response codes.

I don't think this last sentence is necessary, as the other APIs don't say this. Can replace this sentence with: "The API is available through Bot Services." to match other APIs on this doc.  Suggestion below including my other comment.

```suggestion
All users in a meeting receive the notifications sent through the `NotificationSignal` API. The `NotificationSignal` API triggers an in-meeting dialog box and enables you to provide meeting signals that are delivered using the existing conversation notification API for user-bot chat. You can send an in-meeting dialog box based on user action. The API is available through Bot Services.
```